docs/jade-getting-started.md

```diff
@@ -0,0 +1,219 @@
+# Getting Started
+
+> Ensure that your teammates have given you access beforehand to the required
+team resources. If you encounter a permission error, it is likely because you
+are missing appropriate access. Ask for access to Data Biosphere and Google
+Groups. A DevOps colleague will also need to create a Helm `datarepo` definition
+for you.
+
+> These instructions assume you use MacOS, and that you are on the Broad-Internal
+network.
+
+> During this process, you will need your GitHub and Docker Hub username /
+password for multiple steps, so make sure to have those handy. If you don't have
+those yet, see the section below, otherwise you can skip to [Connect Accounts](#2-connect-accounts)
+
+## 1. Create a GitHub and Docker Hub account
+
+GitHub is where the Broad stores our code and projects. Docker Hub allows the
+development team to easily deploy software without having to install lots of
+dependencies.
+
+Sign up to these services with your **personal** email:
+  * https://github.com/join
+  * https://hub.docker.com/signup
+
+## 2. Connect accounts
+
+> Make sure 2-factor authentication (2FA) is activated on your
+[Broad](https://broad.io/2fa) and [GitHub](https://github.com/settings/security)
+account before starting this process!
+
+Connect your GitHub account to your Broad profile:
+
+1. Go to [Broad people](https://people.broadinstitute.org/me) and select the
+**My Profile** tab.
+2. Link your profile to GitHub by clicking under **Other Profiles**.
+3. [Check if the account is successfully linked](https://github.broadinstitute.org/).
+4. Open each of the following GitHub groups and **Request to join** by going to 
+the Members tab: [Broad Institute Read](https://github.com/orgs/broadinstitute/teams/broad-institute-read),
+[Prometheus](https://github.com/orgs/broadinstitute/teams/prometheus),
+[DSDE Engineering](https://github.com/orgs/broadinstitute/teams/dsde-engineering)
+5. To avoid being overwhelmed with notifications, [add your Broad email address](https://github.com/settings/emails),
+[route the notifications](https://github.com/settings/notifications) to that
+email, and [unfollow projects](https://github.com/watching) that are not
+relevant to your team.
+
+Connect your Docker Hub account to your Broad profile by contacting the DevOps
+team.
+
+## 3. Install Homebrew
```

Comment:
would it be worthwhile to make a brewfile that contains everything listed here so it can all be installed with one command? perhaps we can use something like [homebrew-bundle?](https://github.com/Homebrew/homebrew-bundle)

Reply:
A Brewfile would be great.